sui-execution/latest/sui-move-natives/src/test_scenario.rs

```diff
@@ -229,8 +229,17 @@ pub fn end_transaction(
                     .or_default()
                     .insert(id);
             }
-            // TODO: Implement support for ConsensusV2 objects.
-            Owner::ConsensusV2 { .. } => todo!(),
+            Owner::ConsensusV2 { authenticator, .. } => {
+                // Treat ConsensusV2 objects the same as address-owned for now. This will have
+                // to be revisited when other Authenticators are added.
```

Comment:
maybe a debug assert to this affect? Or does `as_single_owner()` panic when there is more than one owner?

Reply:
`as_single_owner` returns a `SuiAddress` directly, not wrapped in an `Option` or anything. so everywhere that uses it will need to be fixed once we add a second `Authenticator` and we need to change the return type of that function.

I added a comment on the function to clarify